misc/style/run-clang-tidy.py

```diff
@@ -82,8 +82,8 @@ def check_search_code_with_clang_tidy():
         "-quiet",
         "-p", build_dir,
         "-clang-tidy-binary=clang-tidy-12",
-        # Include all non-system headers (.*) except the ones from search/ext/.
-        "-header-filter=.*,-tree.hh,-tree_util.hh",
+        # Include all non-system headers.
+        "-header-filter=.*",
```

Comment:
Shouldn't these two files no be unnecessary? Or is the default to ignore all headers (which would be a weird default)?

Reply:
I broke something in `evaluator.h` to test this and ran this with and without the parameter. With the parameter I got the same warning 84 times (once for every path on which the header is included), without the line, I got the warning once.

So the parameter causes a difference in what is checked but since the error still was reported, I'd say it is better to remove the lines.